src/main/java/org/broadinstitute/barclay/help/DocWorkUnitHandler.java

```diff
@@ -56,6 +56,13 @@ public String getDestinationFilename(final DocWorkUnit workUnit) {
     }
 
     /**
+     * Returns the destination prefix (no paths) that the handler would like
```

Comment:
I think this comment is wrong - its the JSON output file name not the destination prefix.